spec/lib/fast_cache/cache_spec.rb

```diff
@@ -112,6 +117,42 @@
         subject[:e].should eq 6
       end
     end
+
+    describe 'cleanup block' do
+      let(:cleanups) { [] }
```

Comment:
I had it that way originally, but felt this was cleaner.  `cleanups` will be initialized the first time _anything_ accesses it, whether the callback or the expectation.  If it's multi-threaded and the expectation gets there before the callback does, then the test is flawed and would fail in any case, even if `cleanups` were initialized in a `before` block.